src/wwwroot/template-editor/index.html

```diff
@@ -15,6 +15,7 @@
 
     <ng-view></ng-view>
     <mseditor-loader class="preloader">
+      <mseditor-modal></mseditor-modal>
```

Comment:
Is it fine adding this element inside the loader? cc: @mvillaseco

Reply:
I have asked @mvillaseco, we agree in that hierarchically it should not be inside `mseditor-loader`, is there any reason to put it there?

If not, maybe between `<ng-view></ng-view>` and `<mseditor-loader class="preloader">` could be a good place.

Reply:
The reason was that the mseditor-modal directive is covering the entire page then when async calls are made from backend any modal window is being showed behind of this element.

Reply:
@mvillaseco what do you think? Is there a more elegant solution?

If it is the only way, please add a comment next to it to help to understand it better and avoid unwanted changes.

---

Also, please think about it, because it can orient some code refactoring in the near future, and an improvement in MSEditor code base: We are showing an error modal in a really lower layer (adapter is the lowest one), it is a smell, it would be great having something to catch expected and unexpected errors in an upper layer. I mean, showing the error and redirect should not be a responsibility of our adapter, it is a responsibility of MSEditor.

It is not important to apply any refactoring right now but take into account, because probably we have to deal with similar cases soon.

Reply:
We decided to left for later this implementation and refactor, and mantain the original behavior of mseditor